patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,17 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Used as a last resort for determining potion effect sort order in the gui.
++     * This can be used to preserve a potion effect's sort order if it has a liquid color that is subject to change.
++     * @param The PotionEffect instance containing the potion
```

Comment:
This is invalid Javadoc. You need to include the parameter name.  Also, it's convention to start this line with a lowercase letter if the first phrase contains no top-level verb. (That is, say `@param something a blah`, not `@param something A blah`, and say `@param something a blah that foos. Used to qux.`, not `@param something A blah that foos. Used to qux`, but say `@param something Does foo. Used to qux.`, not `@param something does foo. Used to qux`). [Reference](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@param)